src/TodoModel.cpp

```diff
@@ -0,0 +1,48 @@
+#include "todo_lib/TodoModel.h"
```

Comment:
add copyrights on the top

Reply:
and also ensure the order of #include. the first should be builtin libraries, the next should be our library